.github/workflows/verify-docker-image.yml

```diff
@@ -0,0 +1,32 @@
+name: Verify Docker Image
+
+on:
+  pull_request:
```

Comment:
@diox to clarify a concern you had about over-pushing. This job runs on pull requests, it loads the image so it can be "tested" in CI. right now a minimal smoke test. It does not and cannot push because you must either load or push, cannot do both with the underlying action.